release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2419 @@
+
+- A task-focused interface.
+- Key call-outs.
+- Unified search.
+- Streamlined navigation.
+
+[[ocp-36-search-catalog]]
+==== Search Catalog (Technology Preview)
+
+The search catalog feature (currently in xref:ocp-36-technology-preview[Technology
+Preview]) provides a single, simple way to quickly get what you want.
+
+image::ocp36-search-catalog.gif[search catalog]
+
+[[ocp-36-add-from-catalog]]
+==== Add from Catalog (Technology Preview)
+
+The add from catalog feature (currently in
+xref:ocp-36-technology-preview[Technology Preview]) allows you to provision a
+service from the catalog.
+
+Select the desired service, then follow prompts for your desired project and
+configuration details.
+
+image::ocp36-add-from-catalog.gif[add from catalog]
+
+[[ocp-36-project-overview-redesign]]
+==== Project Overview Redesign
+
+In {product-title} 3.6, the Project Overview was resigned based on feedback from
+customers.
+
+In this redesign, there are three focused views:
+
+- Applications
+- Pipelines
+- Resource types
+
+There are now more contextual actions and rolled up metrics across multiple
+pods.
+
+image::ocp36-redesigned-project-overview.gif[Redesigned Project Overview]
+
+[[ocp-36-add-to-project]]
+==== Add to Project (Technology Preview)
+
+The add to project feature (currently in
+xref:ocp-36-technology-preview[Technology Preview]) allows you to provision a
+service without having to leave the Project Overview.
+
+When you go directly to the catalog from project, the context is preserved You
+can directly provision, then bind.
+
+image::ocp36-add-to-project.gif[add to project]
+
+[[ocp-36-bind-in-context]]
+==== Bind in Context
+
+The bind in context feature (currently in
+xref:ocp-36-technology-preview[Technology Preview]) allows you to provision a
+service and bind without having to leave the Project Overview.
+
+- Select deployment and initiate a bind.
+- Select from bindable services.
+- Binding is created and the user stays in context
+
+image::ocp36-bind-in-context.gif[bind in context]
+
+[[ocp-36-template-service-broker]]
+==== Template Service Broker (Technology Preview)
+
+The Template Service Broker (currently in
+xref:ocp-36-technology-preview[Technology Preview]) exposes OpenShift templates
+through a Open Service Broker API to the Service Catalog.
+
+The Template Broker matches the lifecycles of provision, deprovision, bind,
+unbind with existing templates. No changes are required to templates, unless you
+expose bind. Your application will get injected with configuration details
+(bind).
+
+[[ocp-36-image-stream-details]]
+==== Image Stream Details
+
+In {product-title} 3.6, additional details are provided about image streams and
+their tags.
+
+This feature leverages Cockpit views from image streams. It details tags and
+provide information about each.
+
+image::ocp36-image-stream-details.png[bind in context]
+
+[[ocp-36-service-catalog-experience]]
+==== Service Catalog Experience in the CLI (Technology Preview)
```

Comment:
No.
a. we're talking different experience. Your `oc cluster up`, which is experience of using CLI to launch a service catalog server env (distinct from using ansible to launch that). My "oc/kubectl catalog ..." talks about experience of using CLI to communicate with already launched service catalog server env.
So it is OK to add "oc cluster up ..." as Service Catalog Experience in the CLI, but not OK to add the "kubectl catalog ..." because they're not implemented in 3.6:
Provision, bind, provision and bind, deprovision, list, and so on
kubectl catalog provision mysql --plan=2cores --namespace=mine
...